client/app/scripts/liveblog-edit/module.js

```diff
@@ -462,10 +462,10 @@ define([
             return privileges.userHasPrivileges({'publish_post': 1});
         }
         function isUserOwner(archive) {
-            if ($rootScope.currentUser._id !== archive.original_creator) {
-                return false;
-            } else {
+            if ($rootScope.currentUser._id === archive.original_creator || $rootScope.currentUser.user_type === 'administrator') {
```

Comment:
@vladnicoara You created a `SecurityService` somewhere, isn't it ? :)
What about using it to check the permission ?